### PR TITLE
Config tweaks to Saucelabs

### PR DESCRIPTION
**Pull request by dhilton**
### What is the context of this PR?

This PR fixes Saucelabs integration with travis and our CI workflow, ensuring that a basic selenium test is run on each PR to ensure our frontend remains consistent. 
### How to review
1. Check out this branch 
2. Clean out your node_modules folder
3. Run npm cache clean
4. Run npm install 
5. Run npm run test_functional to test locally the selenium tests
6. Set two environment variables  (grab the Access key off of @dhilton)

```
export SAUCE_ACCESS_KEY=XXXX
export SAUCE_USERNAME=eqdigital
```
1. Run npm run test_functional_sauce
2. You should now have a test run on SauceLabs testing the MCI survey.
- [ ] Do all commits have sensible commit messages?
- [ ] Is all new code unit tested?
- [ ] Are there integration tests if needed?
- [ ] Is there sufficient logging?
- [ ] Is there documentation or is the code self-describing?
### Who worked on the PR

Both @hamishtaplin and @dhilton worked on this PR.

### CODE REVIEW

**Comment by dhilton** [comment]
Ready for review now pls.

**Comment by warrenbailey** [comment]
LGTM 👍

File: scripts/run_tests.sh [hunk 1]
```diff
@@ -32,12 +38,14 @@ function display_result {
   fi
 }
 
-flake8 --max-complexity 10
```
Comment (collisdigital) [inline scripts/run_tests.sh:35]: Why has this gone?

File: scripts/run_tests.sh [hunk 2]
```diff
@@ -32,12 +38,14 @@ function display_result {
...
+flake8
 display_result $? 1 "Code style check"
-py.test --cov=app $@
```
Comment (collisdigital) [inline scripts/run_tests.sh:37]: Are we removing code coverage or replacing it with an alternative?

File: scripts/run_tests.sh [hunk 3]
```diff
@@ -10,6 +10,12 @@ export EQ_CLOUDWATCH_LOGGING=False
 
 export EQ_RABBITMQ_ENABLED=False
 
+if [ ! -d "/app/data" ]; then
```
Comment (collisdigital) [inline scripts/run_tests.sh:13]: Isn't this and the generate now removed?

**Pull request merged**